apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -272,6 +329,27 @@ type ResourceIdentifier struct {
 	Namespace string `json:"namespace,omitempty"`
 }
 
+// ResourcePlacementStatus represents the status of a group of resource placed on selected cluster
+// according to the PlacementPolicy field in the ClusterResourcePlacement object.
```

Comment:
ResourcePlacementStatus represents the placement status of selected resources for one target cluster.

Remove "according to the PlacementPolicy field in the ClusterResourcePlacement object."